Suggest adt installation (#662)
ansible/ansible-builder: docs/installation.rst

- The ``--container-runtime`` option must correspond to the containerization tool you use.
- ``ansible-builder`` version ``3.x`` requires Python ``3.9`` or higher.

.. note::

   The **recommended** approach to install ``ansible-builder`` is using the

>    ``ansible-dev-tools`` package.

Reviewer: ```suggestion
   Installing ``ansible-builder`` with the ``ansible-dev-tools`` package is a convenient option that offers several benefits over a standalone installation.
```

Reviewer: The original wording seems vague to me. Who recommends installing the ADT package over standalone builder?

>    `Ansible Development Tools (ADT) <https://ansible.readthedocs.io/projects/dev-tools/>`_
>    aims to streamline the setup and usage of several tools needed in order to
>    create `Ansible <https://www.ansible.com>`_ content. ADT combines critical Ansible
>    development packages into a unified Python package.

Reviewer: ```suggestion
   `Ansible Development Tools (ADT) <https://ansible.readthedocs.io/projects/dev-tools/>`_ is a single Python package that contains all the tools you need to create high quality Ansible content.
   The ADT package helps you quickly set up a development environment, generate new collections, and then build and test to ensure consistent and robust automation.
```

Reviewer: I'd get more to the point and tell the reader what ADT is from the start. Also address the reader more directly through the 2nd person singular ("you").

Rather than ADT on its own as an acronym, I'd suggest "the ADT package" to avoid ambiguity.

- The ``--container-runtime`` option must correspond to the containerization tool you use.
- ``ansible-builder`` version ``3.x`` requires Python ``3.9`` or higher.

.. note::

   The **recommended** approach to install ``ansible-builder`` is using the
   ``ansible-dev-tools`` package.
   `Ansible Development Tools (ADT) <https://ansible.readthedocs.io/projects/dev-tools/>`_
   aims to streamline the setup and usage of several tools needed in order to
   create `Ansible <https://www.ansible.com>`_ content. ADT combines critical Ansible
   development packages into a unified Python package.

   .. code-block:: shell

      # This also installs ansible-core if it is not already installed

>       $ pip3 install ansible-dev-tools

Reviewer 2: It might make sense to also fix the other 2 examples of using `pip`, and use the same executable `pip3`, which seems to be the default one (the simple one might be missing on some platforms), same applies to using python.

Author: updated

- The ``--container-runtime`` option must correspond to the containerization tool you use.
- ``ansible-builder`` version ``3.x`` requires Python ``3.9`` or higher.

> Recommended Installation Method

Reviewer 3: you are not really adding a 'recommended install method' .. but pointing to a package that already includes it with it's own installation methods. This is at the very least not well phrased.

********************************

> .. note::

Reviewer 3: make it a note or a header with a paragraph, a header for a note is confusing, looks like someone forgot to populate the text

.. code-block:: shell

   $ pip3 install ansible-builder

.. note::

>    An **alternative** approach to install ``ansible-builder`` is using the ``ansible-dev-tools`` package.

Reviewer: ```suggestion
   An **alternative** approach to installing ``ansible-builder`` is using the ``ansible-dev-tools`` package.
```

>    An **alternative** approach to install ``ansible-builder`` is using the ``ansible-dev-tools`` package.

Reviewer: Do we really want to put "alternative" in bold text? This is already in a note to catch user attention. I'm not sure what emphasizing "alternative" is meant to convey here tbh.

>    An **alternative** approach to installing ``ansible-builder`` is using the ``ansible-dev-tools`` package.
>    `Ansible Development Tools (ADT) <https://ansible.readthedocs.io/projects/dev-tools/>`_ is a single Python package that includes all necessary tools to
>    set up a development environment, generate new collections, build and test the content.

Reviewer 4: ```suggestion
   An **alternative** approach to installing ``ansible-builder`` is using the ``ansible-dev-tools`` package. `Ansible Development Tools (ADT) <https://ansible.readthedocs.io/projects/dev-tools/>`_ is a single Python package that includes all necessary tools to set up a development environment, generate new collections, build and test the content.
```

Just consolidating to a single line, removing some unecessary line breaks

>    set up a development environment, generate new collections, build and test the content consistently, resulting in robust automation.

Reviewer: ```suggestion
   set up a development environment, generate new collections, build and test the content.
```

Reviewer: In the current form it's not clear if "consistently" modifies "build and test the content" or if it also modifies "set up a development environment" and "generate new collections".

If the former, then "consistently build and test the content". If the latter, then "consistently set up a development environment, generated new collections, build and test the content."

Also ", resulting in robust automation" looks like a dangling participle. If you want to keep it, maybe consider ", which results in robust automation."
